example/lib/data_store/meeting_store.dart

```diff
@@ -772,7 +772,7 @@ class MeetingStore extends ChangeNotifier
     }
     if (index != -1) {
       peerTracks[index].stats?.hmsLocalAudioStats = hmsLocalAudioStats;
-      peerTracks[index].notify();
+      if (!peerTracks[index].isOffscreen) peerTracks[index].notify();
```

Comment:
@govindmaheshwari2  can this - isOffscreen check - be handled directly in `PeerTrackNode` class?

Reply:
It is possible, but it will contain many if statements as we are fetching different stats from different callbacks.